docs/deployment/automated-local.rst

```diff
@@ -0,0 +1,227 @@
+	or need to insert additional stars for matching the path.
+
+The "systemd" tag in conjunction with the SYSTEMD_WANTS environment variable has systemd
+launch the "automatic-backup" service, which we will create next, as the
+``/etc/backups/automatic-backup.service`` file:
+
+.. code-block:: ini
+
+    [Service]
+    Type=oneshot
+    ExecStart=/etc/backups/run.sh
+
+Now, create the main backup script, ``/etc/backups/run.sh``. Below is a template,
+modify it to suit your needs (e.g. more backup sets, dumping databases etc.).
+
+.. code-block:: bash
+
+	#!/bin/bash -ue
+	which python borg > /dev/null
+
+	# The udev rule is not terribly accurate and may trigger our service before
+	# the kernel has finished probing partitions. Sleep for a bit to ensure
+	# the kernel is done.
+	#
+	# This can be avoided by using a more precise udev rule, e.g. matching
+	# a specific hardware path and partition.
+	sleep 5
+
+	#
+	# Script configuration
+	#
+
+	# The backup partition is mounted there
+	MOUNTPOINT=/mnt/backup
+
+	# This is the location of the Borg repository
+	TARGET=$MOUNTPOINT/borg-backups/backup.borg
+
+	# Archive name schema.
+	# If you intend to use prune and have more than one backup set, perhaps
+	# swap hostname and date to be compliant with prune's --prefix option.
+	DATE=$(date --iso-8601)-$(hostname)
+
+	# This is the file that will later contain UUIDs of registered backup drives
+	DISKS=/etc/backups/backup.disks
+
+	# Find whether the connected block device is a backup drive
+	for uuid in $(lsblk -nlpo uuid)
+	do
+	        if grep -qF $uuid $DISKS; then
+	                break
+	        fi
+	        uuid=
+	done
+
+	if [ ! $uuid ]; then
+	        echo "No backup disk found, exiting"
+	        exit 0
+	fi
+
+	echo "Disk $uuid is a backup disk"
+	disk_path=/dev/disk/by-uuid/$uuid
+	# Mount file system if not already done. This assumes that if something is already
+	# mounted at $MOUNTPOINT, it is the backup drive. It won't find the drive if
+	# it was mounted somewhere else.
+	(mount | grep $MOUNTPOINT) || mount $disk_path $MOUNTPOINT
+	drive=$(lsblk -snlpo name $disk_path | head -n1)
+	echo "Drive path: $drive"
+
+	#
+	# Create backups
+	#
+
+	# Options for borg create
+	BORG_OPTS="-svx --compression lz4 --checkpoint-interval 86400"
+
+	# Set BORG_PASSPHRASE or BORG_PASSCOMMAND somewhere around here, using export,
+	# if encryption is used.
+
+	# No one can answer if Borg asks these questions, it is better to just fail quickly
+	# instead of hanging.
+	export BORG_RELOCATED_REPO_ACCESS_IS_OK=no
+	export BORG_UNKNOWN_UNENCRYPTED_REPO_ACCESS_IS_OK=no
+
+	# Log Borg version
+	borg -V
+
+	echo "Starting backup for $DATE"
+
+	borg create $BORG_OPTS \
+	  --exclude /root/.cache \
+	  --exclude /var/cache \
+	  --exclude /var/lib/docker/devicemapper \
+	  $TARGET::$DATE-$$-system \
+	  / /boot
+
+	borg create $BORG_OPTS \
+	  --exclude 'sh:/home/*/.cache' \
+	  $TARGET::$DATE-$$-system \
+	  /home/
```

Comment:
if this backups /home the previous one needs to exclude home explicitly and not depend on -x doing this.